build_stage1.sh

```diff
@@ -70,6 +70,10 @@ ostree remote add --repo=/srv/repo centos-atomic-host --set=gpg-verify=false htt
 
 rpm-ostree compose --repo=${OstreeRepoDir} tree ${GitDir}/centos-atomic-host.json |& tee ${BuildDir}/log.compose
 
+## stop the script unless a new commit has been made
+
+cat ${BuildDir}/log.compose | grep -q "Wrote commit"
```

Comment:
It would probably more `rpm-ostree`-like to use the `--touch-if-changed` option when doing `rpm-ostree compose tree`

Requires a little more logic, but is probably better than grepping the log.

Reply:
See an example of that here: https://github.com/CentOS/sig-atomic-buildscripts/blob/0e99976b9990afa298d51e198a87ed6aeba2a024/centos-ci/run-treecompose#L25

Reply:
OK, awesome, thanks.